src/metldata/builtin_transformations/infer_relation/assumptions.py

```diff
@@ -21,19 +21,21 @@
     assert_path_classes_and_relations_exist,
     assert_relation_does_not_exist,
 )
-from metldata.builtin_transformations.infer_relation.config import InferRelationConfig
+from metldata.builtin_transformations.common.path.path import RelationPath
```

Comment:
This could also be refactored/renamed, `common.path.path` is a bit of a code smell.